### PR TITLE
Rewrite decrypt_all_files_and_variables to Bash

### PR DESCRIPTION
**Pull request by mik-laj**
Nadal WIP, ale jestem otwarty na sugestię, czy to ma sens.

### CODE REVIEW

**Comment by potiuk** [comment]
Yep. it makes sense :)

**Comment by potiuk** [comment]
Continue and finish please

**Comment by mik-laj** [comment]
@potiuk  I finished my changes. Do you have any comments?

File: run_environment.sh [hunk 1]
```diff
@@ -210,7 +210,7 @@ decrypt_all_variables() {
     echo "Decrypting encrypted variables"
     echo
     (set -a && source "${GCP_CONFIG_DIR}/variables.env" && set +a && \
-     python ${MY_DIR}/_decrypt_encrypted_variables.py ${AIRFLOW_BREEZE_PROJECT_ID} >\
+     python ${MY_DIR}/_decrypt_encrypted_variables.sh ${AIRFLOW_BREEZE_PROJECT_ID} >\
```
Comment (potiuk) [inline run_environment.sh]: This won't work :). -> python + .sh  ? Can you double check that ?

Comment (mik-laj) [reply run_environment.sh]: My mistake. I fixed. After executing the command `./run_environemt.sh`, the environment gets up

**Review by potiuk — APPROVED** [review]
LGTM. Works for me as well.

**Pull request merged**